Tzkt.Api/appsettings.json

```diff
@@ -6,6 +6,7 @@
   "ResponseCache": {
     "CacheSize": 256
   },
+  "MaxAttemptsForMigrations": 30,
```

Comment:
Frankly speaking, I'd just increase the hard-coded constant (for instance, to `60`), to avoid overcomplicating configuration, because this setting is not really needed, in my opinion (I don't think anyone will ever want to set any specific value other than '999999'😁).
Or, if you think `60` will also be not enough, check this https://github.com/baking-bad/tzkt/issues/137#issuecomment-1372092631

Reply:
Is it OK if I just increase it to 120? That worked well for us.

Alternatively, as the `MaxAttemptsForMigrations` has a sane default, we can remove it from `appSettings.json` and only someone who needs it can add it / override it in Env Vars,

Reply:
Yep, let's set 120 for now.
I have some thoughts on how to improve the API initialization (so there will be no exceptions if it's not a "dead-end", like incompatible migrations), so I wouldn't want to introduce additional settings which might be deprecated in some short time :)